orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTask.java

```diff
@@ -45,14 +46,17 @@
 
   private final OortService oortService;
 
+  @Value("${kubenetes.manifest.timeout-minutes:30}")
```

Comment:
Please use constructor injection instead of value injection, and make this field `final`.  You should be able to just annotate the appropriate parameter of the constructor with this `@Value` annotation.

Also, it looks like a number of other places that have added overridable timeouts have scoped them as `tasks.task-name-timeout`...though arguably it might even be better as `tasks.task-name.timeout-minutes`.  At least I think it would be best to have this scoped under `tasks` to avoid too many top-level things.

The other configurable timeouts also specify milliseconds...for consistency, maybe it's better for these to all be the same?  (Though I do see a point that people probably don't need that precision...)